model/src/gateway/payload/voice_state_update.rs

```diff
@@ -157,4 +157,80 @@ mod tests {
             ],
         );
     }
+
+    #[test]
+    fn voice_state_update_deser() {
+        let input = serde_json::json!({
```

Comment:
All new tests should use [`serde_test`](https://docs.rs/serde_test/1.0.114/serde_test/), `serde_json` in tests is being phased out since it doesn't cleanly test for equivalence. You can see a [reaction test example here](https://github.com/twilight-rs/twilight/blob/244259a44b39d934d16bb9b2f761bd28d8bd0bff/model/src/channel/reaction.rs#L154-L286).

Reply:
You can see a [recent PR](https://github.com/twilight-rs/twilight/pull/266), #266, as an example of just switching over a test to use `serde_test` too.